packages/microcosm-www-next/src/components/graphic.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react'
 
-const Graphic = ({ section, graphicUrl }) => (
+const Graphic = ({ section, microcosmView }) => (
   <figure
     id={'graphic-' + section}
     className="section__graphic__figure"
     data-module="ObserveGraphic"
     data-section={section}
   >
-    <img src={graphicUrl} alt="TODO" />
+    <img
+      src={
+        microcosmView ? `/${section}-microcosm.png` : `/${section}-browser.png`
+      }
+      alt="TODO"
```

Comment:
?

Reply:
I'll replace all the `alt="TODO"` at the end